source/_integrations/qrcode.markdown

```diff
@@ -39,3 +39,21 @@ source:
       required: false
       type: string
 {% endconfiguration %}
+
+## Basic example
+
+An automation using this integration could look like this:
+
+```yaml
+automation:
+  - alias: 'Catch QR code and arm the alarm system'
+    trigger:
+      platform: state
+      entity_id: image_processing.qr_front_door
+      to: !secret ARM_QR_CODE_VALUE
+    action:
+      service: alarm_control_panel.alarm_arm_away
```

Comment:
This needs to be a list, see our YAML style guide: <https://developers.home-assistant.io/docs/documenting/yaml-style-guide>

```suggestion
      - service: alarm_control_panel.alarm_arm_away
```